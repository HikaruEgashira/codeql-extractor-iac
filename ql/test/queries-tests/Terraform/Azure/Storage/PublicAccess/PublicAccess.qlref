security/Terraform/Azure/ManagedDisk/PublicAccess.ql