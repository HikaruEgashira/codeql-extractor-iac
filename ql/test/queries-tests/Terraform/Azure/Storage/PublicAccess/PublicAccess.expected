| storage.tf:9:1:15:1 | resource azurerm_storage_container insecure | Azure Storage is Unencrypted for 'insecure' |
